container/shim/src/modules/tls.js

```diff
@@ -13,6 +13,7 @@ export const BACKUP_CERT_PATH = `${SSL_PATH}/node.backup.crt`;
 export const BACKUP_KEY_PATH = `${SSL_PATH}/node.backup.key`;
 
 export const certExists = await fsPromises.stat(CERT_PATH).catch((_) => false);
+export const backupCertExists = await fsPromises.stat(BACKUP_CERT_PATH).catch((_) => false);
```

Comment:
This isn't used anywhere, is that intended?

Reply:
Yes, I had included it to conditionally do our logic, but I want the logs. Thing is, not everyone has a backup cert yet, and we can only swap certs once they have them, so this variable will be used.